packages/transformers/elm/src/ElmTransformer.js

```diff
@@ -0,0 +1,125 @@
+import {Transformer} from '@parcel/plugin';
+import commandExists from 'command-exists';
+import path from 'path';
+import {minify} from 'terser';
+import ThrowableDiagnostic from '@parcel/diagnostic';
+
+export default new Transformer({
+  async transform({asset, options}) {
+    const elm = await setupElmCompiler(asset, options);
+
+    const config = {
+      cwd: path.dirname(asset.filePath),
+      debug: !options.env.PARCEL_ELM_NO_DEBUG && options.mode !== 'production',
+      optimize: asset.env.minify,
+    };
+    asset.invalidateOnEnvChange('PARCEL_ELM_NO_DEBUG');
+    ResolveDependenciesFromAsset(elm, asset);
```

Comment:
Ideally run the for loop directly in `transform`, or at least rename it to be lower case. This is not a class